pkgs/by-name/co/cosmic-calculator/package.nix

```diff
@@ -0,0 +1,91 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  rustPlatform,
+  makeBinaryWrapper,
+  pkg-config,
+  wayland,
+  libxkbcommon,
+  just,
+  cosmic-icons,
+}:
+let
+  commitDate = "2024-09-07";
+in
+rustPlatform.buildRustPackage rec {
+  pname = "cosmic-calculator";
```

Comment:
The package name is `cosmic-ext-calculator` in upstream, so I think this should be renamed to it